login/login.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>로그인페이지</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="../style.css">
+</head>
+<body>
+    <main>
+        <img class="login-logo" src="./images/loginLogo.png" alt="로그인탑이미지">
+
+        <form class="login-form">
+            <label class="login-form-label" for="email">이메일</label>
+            <input class="login-form-input" type="text" placeholder="이메일">

```

Comment:
❗️ 수정요청
label 태그를 사용하신 것은 좋습니다. 다만 input과 연결을 해주셔야해요. for 속성에는 연결하고자하는 input의 id가 들어가야합니다~

```suggestion
            <label class="login-form-label" for="email">이메일</label>
            <input id="email" class="login-form-input" type="text" placeholder="이메일">
```